src/cli.js

```diff
@@ -86,16 +86,6 @@ class CLI {
         description: 'Specifies the action kind.',
         default: 'nodejs:10',
       })
-      .option('web-export', {
-        description: 'Annotates the action as web-action',
-        type: 'boolean',
-        default: true,
-      })
-      .option('raw-http', {
```

Comment:
for web-action, we could make raw the default, but it still needs to be configurable.